src/widgets/Finance/components/Finance.js

```diff
@@ -14,27 +19,31 @@ const updatedTime = updated => {
   return null;
 };
 
-const Finance = ({ finance }) =>
-  <div className="panel Number">
-    <div className="panel-heading">
-      {finance.finance.company} Stock Price
-    </div>
-    <div className="panel-body">
-      <span className="current">
-        {finance.finance.price}
-      </span>
-    </div>
-    <div className="panel-footer">
-      {finance.finance.change > 0
-        ? <span className="change increase good">
-          {finance.finance.change}
+const Finance = ({ connectDragSource, connectDropTarget, isDragging, isOver, ...props }) =>
+  compose(connectDragSource, connectDropTarget)(
+    <div className="panel Number" style={{ opacity: isDragging || isOver ? 0.5 : 1 }}>
+      <div className="panel-heading">
+        {props.finance.finance.company} Stock Price
+      </div>
+      <div className="panel-body">
+        <span className="current">
+          {props.finance.finance.price}
         </span>
-        : <span className="change decrease bad">
-          {finance.finance.change}
-        </span>}
-      {updatedTime(finance.finance.updated)}
-    </div>
-  </div>;
+      </div>
+      {/* eslint-disable react/jsx-indent */}
```

Comment:
So this is another possible workaround to the linter messing up the syntax? Interesting. I factored this section out to its own method instead on a separate branch. This might not be needed after rebase.